src/App.native.tsx

```diff
@@ -51,12 +54,12 @@ function InnerApp() {
   useEffect(() => {
     notifications.init(queryClient)
     listenSessionDropped(() => {
-      Toast.show('Sorry! Your session expired. Please log in again.')
+      Toast.show(_(msg`Sorry! Your session expired. Please log in again.`))
```

Comment:
I don't think we would want this string to be localized as localizing it will have a lot of performance implications since this is the root component and it'll be re-rendered

Reply:
So I think let's remove this localization

Reply:
Okay, I understood.